azkaban-db/src/main/java/azkaban/db/DatabaseSetup.java

```diff
@@ -23,59 +23,35 @@
 import java.io.IOException;
 import java.sql.Connection;
 import java.sql.SQLException;
-import java.util.HashMap;
 import java.util.HashSet;
-import java.util.Map;
 import java.util.Set;
 import org.apache.commons.dbutils.QueryRunner;
 import org.apache.commons.io.IOUtils;
 import org.apache.log4j.Logger;
 
-/**
- * TODO kunkun-tang: this class is quite messy now, needs to be refactored.
- */
 public class DatabaseSetup {
 
-  public static final String DATABASE_SQL_SCRIPT_DIR =
-      "database.sql.scripts.dir";
   private static final Logger logger = Logger
       .getLogger(DatabaseSetup.class);
-  private static final String DEFAULT_SCRIPT_PATH = "sql";
   private static final String CREATE_SCRIPT_PREFIX = "create.";
   private static final String SQL_SCRIPT_SUFFIX = ".sql";
 
-  private static final String INSERT_DB_PROPERTY =
-      "INSERT INTO properties (name, type, value, modified_time) values (?,?,?,?)";
-  private static final String UPDATE_DB_PROPERTY =
-      "UPDATE properties SET value=?,modified_time=? WHERE name=? AND type=?";
-
   private final AzkabanDataSource dataSource;
-  private final Set<String> missingTables = new HashSet<>();
-  private final Map<String, String> installedVersions = new HashMap<>();
-  private String version;
-  private boolean needsUpdating;
-
+  private final Set<String> tables = new HashSet<>();
```

Comment:
i don't think tables as member variable is necessary.

Reply:
Should be right.

Reply:
+1